gradle/versions.gradle

```diff
@@ -0,0 +1,39 @@
+// Variables for entire project
+ext {
+  // Setup
+  minSdkVersion = 14
+  targetSdkVersion = 25
+  compileSdkVersion = 25
+  buildToolsVersion = "25.0.0"
+
+  // Common versions
+  supportLibraryVersion = "25.0.0"
+}
+
+// Plugins
+ext.plugs = [
+    "gradle"                  : "com.android.tools.build:gradle:2.2.3",
+    "androidMavenGradlePlugin": "com.github.dcendents:android-maven-gradle-plugin:1.4.1"
+]
+
+// Dependencies
+ext.deps = [
```

Comment:
What are you asking?

`plugs` is for Gradle plugins only - `buildscript` block
`deps` is for Gradle dependencies only - `dependencies` block

It is a clear and definite separation. Do you have a suggestion here?